test/withStateMachine.spec.js

```diff
@@ -105,6 +111,7 @@ test('state', () => {
 test('actions', () => {
```

Comment:
I wouldn't repurpose this test, but I suggest we create a separate one (like "invalid transition with extended state") or, add a new case to "render".